test/NuGet.Core.FuncTests/Dotnet.Integration.Test/Dotnet.Integration.Test.csproj

```diff
@@ -28,4 +24,4 @@
   </ItemGroup>
   <Import Project="$(BuildCommonDirectory)common.targets" />
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
-</Project>
+</Project>
```

Comment:
your text editor didn't honor our `.editorconfig` setting to end the file with a newline.